### ISSUE TITLE
TensorFlow Lite in C++ doesn't support the TensorFlow operation of Dropout

### ISSUE THREAD

**Issue by twister11211** [TensorSpeech/TensorFlowTTS#703]
In the CPP tflite example it says the inference function need delete Dropout before converting tflite model.
I have opened the colab after cloning the repository in tensorflow_tts/models/fastspeech2.py I changed training to false on both lines but the runtime crashes, I have also tried deleting the dropout from here like the example describes but it still needs tf.lite.OpsSet.SELECT_TF_OPS

f0_embedding = self.f0_dropout(
                                                     self.f0_embeddings(tf.expand_dims(f0_outputs, 2)), training=False
                                                     )
energy_embedding = self.energy_dropout(
                                                                    self.energy_embeddings(tf.expand_dims(energy_outputs, 2)), training=False
                                                                    ) 

f0_embedding = self.f0_embeddings(tf.expand_dims(f0_outputs, 2))
energy_embedding = self.energy_embeddings(tf.expand_dims(energy_outputs, 2))

Is this the correct way to do this or do I have it wrong

**Comment by dathudeptrai** [comment]
@twister11211 the last time i tried this model on tensorflow lite was successfully (just need to remove dropout layer). Maybe there is a bug come from tflite itself rather than the code from this repo.

**Comment by twister11211** [comment]
@dathudeptrai  would you be able to confirm the steps for me so I can focus my energy in the correct place
1: open the colab in the C++ example
https://colab.research.google.com/drive/1Ma3MIcSdLsOxqOKcN1MlElncYMhrOg3J?usp=sharing#scrollTo=KCm6Oj7iLlu5
2: run the first cell
3: open the file /content/TensorFlowTTS/tensorflow_tts/models/fastspeech2.py
4: delete the dropout on lines 254 to 259
5: run the rest of the colab

I'm able to delete the fastspeech2.py file and the conversion runs the same so I'm not certain if editing it effects anything
